conftest.py

```diff
@@ -77,6 +77,6 @@ def experiment(environ):  # pylint: disable=redefined-outer-name,unused-argument
     """Mock an experiment."""
     os.environ['WORK'] = '/work'
     os.environ['EXPERIMENT'] = 'test-experiment'
-    os.environ['CLOUD_EXPERIMENT_BUCKET'] = 'gs://experiment-data'
-    os.environ['CLOUD_WEB_BUCKET'] = 'gs://web-bucket'
+    os.environ['EXPERIMENT_FILESTORE'] = 'gs://experiment-data'
+    os.environ['REPORTS_FILESTORE'] = 'gs://web-bucket'
```

Comment:
@lszekeres @inferno-chromium Since we're changing this API do you think it makes sense to call it "reports_filestore" or "report_filestore" given that the "experiment_filestore" is singular?

Reply:
report_filestore seems better since this is in the context of this one report generation.

Reply:
I can do this change in this PR.

Reply:
Done.

Reply:
Good point! We store the data of multiple experiments in the former and multiple reports in the latter, right? Singular might suggest otherwise, but having consistent names would be good indeed. So either experiments/reports_filestore, or experiment/report_filestore.

Reply:
(sorry sent this from on old tab which didn't show the discussion above, but LGTM)

Reply:
I Sorta agree about both plural being ideal, but I think it doesn't matter as long as they are consistent.